src/PersonObjects/Person.ts

```diff
@@ -31,35 +35,44 @@ export abstract class Person implements IPerson {
     intelligence: 0,
   };
 
-  mults = defaultMultipliers();
-
-  /** Augmentations */
-  augmentations: PlayerOwnedAugmentation[] = [];
-  queuedAugmentations: PlayerOwnedAugmentation[] = [];
+// Multipliers affecting various actions and attributes, represents skills gained and new speeds / rates because of these skills.
+mults = defaultMultipliers();
 
-  /** City that the person is in */
-  city: CityName = CityName.Sector12;
+/** Augmentations */
+augmentations: PlayerOwnedAugmentation[] = []; // List of augmentations owned by the person
+queuedAugmentations: PlayerOwnedAugmentation[] = []; // List of augmentations queued for installation by the person
 
-  gainHackingExp = personMethods.gainHackingExp;
-  gainStrengthExp = personMethods.gainStrengthExp;
-  gainDefenseExp = personMethods.gainDefenseExp;
-  gainDexterityExp = personMethods.gainDexterityExp;
-  gainAgilityExp = personMethods.gainAgilityExp;
-  gainCharismaExp = personMethods.gainCharismaExp;
-  gainIntelligenceExp = personMethods.gainIntelligenceExp;
-  gainStats = personMethods.gainStats;
-  regenerateHp = personMethods.regenerateHp;
-  queryStatFromString = personMethods.queryStatFromString;
-  updateSkillLevels = personMethods.updateSkillLevels;
-  hasAugmentation = personMethods.hasAugmentation;
-  calculateSkill = calculateSkill; //Class version is equal to imported version
+/** City that the person is in, Sector12 by default */
+city: CityName = CityName.Sector12;
 
-  /** Reset all multipliers to 1 */
-  resetMultipliers() {
-    this.mults = defaultMultipliers();
-  }
+// Methods that increment experience levels towards each skill type.
+gainHackingExp = personMethods.gainHackingExp;
+gainStrengthExp = personMethods.gainStrengthExp;
+gainDefenseExp = personMethods.gainDefenseExp;
+gainDexterityExp = personMethods.gainDexterityExp;
+gainAgilityExp = personMethods.gainAgilityExp;
+gainCharismaExp = personMethods.gainCharismaExp;
+gainIntelligenceExp = personMethods.gainIntelligenceExp;
+gainStats = personMethods.gainStats;
+regenerateHp = personMethods.regenerateHp;
+queryStatFromString = personMethods.queryStatFromString;
+updateSkillLevels = personMethods.updateSkillLevels;
+hasAugmentation = personMethods.hasAugmentation;
+calculateSkill = calculateSkill; //Class version is equal to imported version
 
-  abstract takeDamage(amt: number): boolean;
-  abstract whoAmI(): string;
-  abstract toJSON(): IReviverValue;
+/** Reset all multipliers to 1 */
+resetMultipliers() {
+  this.mults = defaultMultipliers();
 }
+
+// Abstract method for taking damage amount amt. Returns true if person is alive after damage and false otherwise.
```

Comment:
For methods, you should use JSDoc syntax if you're going to document them. https://jsdoc.app/about-getting-started
Don't feel the need to go overboard or document every parameter.

You don't need to mention "abstract", that's already part of the signature.

This function is a good use of comment: The return value is non-obvious.